Khan/perseus
# Fix pills so they allow for multiline answers

## Summary

The latest update to Wonder Blocks Pill give it a set height. This makes it cut off multi-line answers in the answer pills in the label-image widget.

To fix this, I added `height: "auto"` to the pill style to reset its height and stop the clipping.

Issue: LC-1664

## Test plan:

- Comment out the added `height: "auto"`
- Go to label-image widget storybook
  - http://localhost:6006/?path=/story/perseus-widgets-label-image--label-image-numberline
- Click on a number line dot and select any of the options.
- Confirm that the answer pill is cut off.
- Add back the `height: "auto"`
- Go back to the number line and confirm that the answer pill is fully displayed now.

### Before

<img width="387" alt="Screenshot 2024-01-25 at 1 56 17 PM" src="https://github.com/Khan/perseus/assets/13231763/0621b36e-2fe1-4944-97b4-5c77666a3012">

### After

<img width="388" alt="Screenshot 2024-01-25 at 1 56 31 PM" src="https://github.com/Khan/perseus/assets/13231763/ebd54db7-3df8-4571-b73c-4650d987e3a0">
